src/components/CalendarMonth.jsx

```diff
@@ -189,7 +192,11 @@ class CalendarMonth extends React.Component {
         </div>
 
         <table
-          {...css(styles.CalendarMonth_table)}
+          {...css(
+            !verticalSpacing && styles.CalendarMonth_table,
+            verticalSpacing && styles.CalendarMonth_verticalSpacing,
+            { borderSpacing: `0px ${verticalSpacing}px` },
```

Comment:
Does this get weird when the verticalSpacing is undefined? Like it would try to apply a style of `0px undefinedpx` and then it would crap out. I think it would be better to do:
```
verticalSpacing && {
  borderCollapse: 'separate',
  borderSpacing: `0px ${verticalSpacing}px`,
},
```
or
```
verticalSpacing && styles.CalendarMonth_verticalSpacing,
verticalSpacing && { borderSpacing: `0px ${verticalSpacing}px` },
```